csharp/src/Ice/OutgoingResponseFrame.cs

```diff
@@ -29,6 +29,8 @@ public sealed class OutgoingResponseFrame
         /// <summary>The frame byte count.</summary>
         public int Size { get; private set; }
 
+        // The compression status from the incoming request.
```

Comment:
Should we make it public?

Reply:
Same as above... it's considered an implementation detail only used for ice1.